lib/archethic/transaction_chain.ex

```diff
@@ -1068,7 +1068,7 @@ defmodule Archethic.TransactionChain do
   def proof_of_integrity([tx = %Transaction{} | _]) do
     tx
     |> Transaction.to_pending()
-    |> Transaction.serialize()
+    |> Transaction.serialize(:extended)
```

Comment:
@samuelmanzanera 
Because I change the POI calculation, I have a choice to make. Either: 

- change `protocol_version`
- or I could use a different flag than the `serialization_mode`. (Ex: `Transaction.serialize([recipient_args: :compact, code: :extended])`)

what do you think?